miso-web/src/main/webapp/scripts/hot.js

```diff
@@ -459,6 +458,16 @@ var HotUtils = {
     });
   },
   
+  toFlatObj : function(item, columns) {
```

Comment:
Can you make this function locally-scoped? If it's not safe for external entities to call it, it shouldn't be visible.